src/main/resources/init.sql

```diff
@@ -5,9 +5,15 @@ CREATE TYPE ticket_type AS ENUM (
     'YEAR'
 );
 
+CREATE TYPE user_status AS ENUM (
+     'ACTIVATED',
+     'DEACTIVATED'
+);
+
 CREATE TABLE user_data (
     id uuid PRIMARY KEY NOT NULL UNIQUE,
     name character varying NOT NULL,
+    user_status public.user_status NOT NULL,
```

Comment:
no need to use `public.`:
`user_status user_status NOT NULL,`
and
`ticket_type ticket_type NOT NULL,`

Reply:
Deleted  public.